### PR TITLE
LPS-137293 | master

### PR DESCRIPTION
**Pull request by zhangyan0701**
https://issues.liferay.com/browse/LPS-137293

### CODE REVIEW

**Comment by john-co** [comment]
ci:test:frontend

**Comment by john-co** [comment]
ci:forward

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase [hunk 1]
```diff
@@ -191,6 +191,13 @@ definition {
 			groupName = "WYSIWYG Site Name",
 			title = "Web Content Title");
 
+		JSONDocument.addFileWithUploadedFile(
+			dmDocumentDescription = "DM Document Description",
+			dmDocumentTitle = "Document_2.jpg",
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase:196]: Not sure if this is a typo, Document_1.jpg? It doesn't affect the test but just looks inconsistent.

Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase:196]: Looks trivial enough to forward now and update this later if needed.

**Comment by john-co** [comment]
lgtm!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase (hunk 1 above)
Comment (zhangyan0701) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase:196]: Yes, that's a typo, I will fix it later.

**Comment by zhangyan0701** [comment]
Resend to BChan because rebase error:
https://github.com/brianchandotcom/liferay-portal/pull/105596

**Pull request closed without merging**